src/test/java/com/facebook/openwifirrm/optimizers/TestUtils.java

```diff
@@ -34,6 +35,8 @@ public class TestUtils {
 
 	/** Default channel width in MHz */
 	public static final int DEFAULT_CHANNEL_WIDTH = 20;
+	/** Default tx power in dBm */
+	public static final int DEFAULT_TX_POWER = 20;
```

Comment:
I believe we have some `DEFAULT_TX_POWER` somewhere else. It's good to keep them consistent if we can. Try reusing the existing default?

Reply:
It depends on whether we want those "default" to be identical logically. If not, I'm ok with keep them separate.